docs/builtins.rst

```diff
@@ -0,0 +1,217 @@
+================
+Builtins in L20n
```

Comment:
Let's use the word `Function` to describe the general class of these things, and then `Built-in Function` to describe those that are provided by `MessageContext` by default.